C/casacore/build_tarballs.jl

```diff
@@ -0,0 +1,89 @@
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+OPENBLAS=(-lopenblas64_)
+
+cd $WORKSPACE/srcdir/casacore
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_PYTHON=OFF \
+      -DBUILD_SHARED_LIBS=ON \
+      -DUSE_READLINE=OFF \
+      -DUSE_HDF5=ON \
+      -DFFTW3_DISABLE_THREADS=ON \
+      -DBLAS_LIBRARIES="${OPENBLAS[*]}" \
+      ..
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcasa_casa", :libcasa_casa),
+    LibraryProduct("libcasa_coordinates", :libcasa_coordinates),
+    LibraryProduct("libcasa_derivedmscal", :libcasa_derivedmscal),
+    LibraryProduct("libcasa_fits", :libcasa_fits),
+    LibraryProduct("libcasa_images", :libcasa_images),
+    LibraryProduct("libcasa_lattices", :libcasa_lattices),
+    LibraryProduct("libcasa_meas", :libcasa_meas),
+    LibraryProduct("libcasa_measures", :libcasa_measures),
+    LibraryProduct("libcasa_mirlib", :libcasa_mirlib),
+    LibraryProduct("libcasa_msfits", :libcasa_msfits),
+    LibraryProduct("libcasa_ms", :libcasa_ms),
+    LibraryProduct("libcasa_scimath_f", :libcasa_scimath_f),
+    LibraryProduct("libcasa_scimath", :libcasa_scimath),
+    LibraryProduct("libcasa_tables", :libcasa_tables),
+    ExecutableProduct("casahdf5support", :casahdf5support),
+    ExecutableProduct("findmeastable", :findmeastable),
+    ExecutableProduct("fits2table", :fits2table),
+    ExecutableProduct("image2fits", :image2fits),
+    ExecutableProduct("imagecalc", :imagecalc),
+    ExecutableProduct("imageregrid", :imageregrid),
+    ExecutableProduct("imageslice", :imageslice),
+    ExecutableProduct("lsmf", :lsmf),
+    ExecutableProduct("measuresdata", :measuresdata),
+    ExecutableProduct("measuresdata-update", :measuresdata_update),
+    ExecutableProduct("ms2uvfits", :ms2uvfits),
+    ExecutableProduct("msselect", :msselect),
+    ExecutableProduct("readms", :readms),
+    ExecutableProduct("showtableinfo", :showtableinfo),
+    ExecutableProduct("showtablelock", :showtablelock),
+    ExecutableProduct("tablefromascii", :tablefromascii),
+    ExecutableProduct("taql", :taql),
+    ExecutableProduct("tomf", :tomf),
+    ExecutableProduct("writems", :writems),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"))
+    Dependency(PackageSpec(name="LAPACK_jll", uuid="51474c39-65e3-53ba-86ba-03b1b862ec14"))
+    Dependency(PackageSpec(name="Bison_jll", uuid="0f48145f-aea8-549d-8864-7f251ac1e6d0"))
```

Comment:
We generally use bison as a `HostBuildDependency`, is it the case also here?

Reply:
It seems to be building without explicitly having a Bison dep, so I'll just remove it for now